### ISSUE TITLE
[Source, Portal2] Certain model animations in specific maps do not play properly

### ISSUE THREAD

**Issue by pivotman319-owo** [magcius/noclip.website#557]
As described above; models like intro1's container ride debris and the headless player arms in finale4 are affected. This is likely the result of incompatible animation data being fed through the renderer.

![Invalid animation data breaking original model's rotation](https://user-images.githubusercontent.com/15126754/207414570-dd954cb8-3890-40cd-b8e7-96ea2dd7e67a.png)

Invoking the following in sp_a1_intro1:

```js
main.viewer.scene.bspRenderers[0].entitySystem.fireInput(main.viewer.scene.bspRenderers[0].entitySystem.entities[326].input_trigger(main.viewer.scene.bspRenderers[0].entitySystem))
```

will trigger the container ride sequence, but the renderer will not be able to read incoming animation data from the sequence itself.

**Comment by magcius** [comment]
Yeah, I can definitely believe it won't function correctly, a lot of the sequence/animation playback system was put together very quickly.

Would love some help!
